third_party/terraform/tests/resource_compute_network_test.go

```diff
@@ -168,6 +176,32 @@ func testAccCheckComputeNetworkExists(t *testing.T, n string, network *compute.N
 	}
 }
 
+func testAccCheckComputeNetworkDefaultRoutesDeleted(t *testing.T, n string, network *compute.Network) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		if rs.Primary.Attributes["name"] == "" {
+			return fmt.Errorf("No ID is set")
+		}
+
+		config := googleProviderConfig(t)
+
+		routes, err := config.clientCompute.Routes.List(config.Project).Filter(fmt.Sprintf("(network=\"%s\") AND (destRange=\"0.0.0.0/0\")", network.SelfLink)).Do()
```

Comment:
This test is failing due this check to not having permissions in CI. I think its because `config.Project` may not be what we want here. Maybe try getting the test project using `getTestProjectFromEnv`?

Reply:
Oh I was looking at the old version. Nevermind